packages/@apphosting/adapter-nextjs/src/bin/build.spec.ts

```diff
@@ -13,18 +13,18 @@ describe("build commands", () => {
   beforeEach(() => {
     tmpDir = generateTmpDir();
     outputBundleOptions = {
-      bundleYamlPath: path.join(tmpDir, ".apphosting/bundle.yaml"),
+      bundleYamlPath: path.join(tmpDir, ".apphosting", "bundle.yaml"),
       outputDirectoryBasePath: path.join(tmpDir, ".apphosting"),
-      outputDirectoryAppPath: path.join(tmpDir, ".apphosting"),
-      outputPublicDirectoryPath: path.join(tmpDir, ".apphosting/public"),
-      outputStaticDirectoryPath: path.join(tmpDir, ".apphosting/.next/static"),
-      serverFilePath: path.join(tmpDir, ".apphosting/server.js"),
+      outputDirectoryAppPath: path.join(tmpDir, ".next", "standalone"),
```

Comment:
why are these output paths changed to ".next/standalone"?

Reply:
We decided during the api review for the new spec to only have bundle.yaml in .apphosting. We will no longer move all the build assets into .apphosting